projects/igniteui-angular/src/lib/carousel/carousel.component.ts

```diff
@@ -64,78 +79,112 @@ export class IgxCarouselComponent implements OnDestroy {
      * Sets whether the carousel can pause the slide transitions.
      *
      * Default value is true
-     *
+     * ```html
+     *  <igx-carousel [pause]="false"></igx-carousel>
+     * ```
      * @memberOf IgxCarouselComponent
      */
     @Input() public pause = true;
-
+    /**
+     *Returns the time interval in milliseconds before the slide changes.
+     *```typescript
+     * let timeInterval = this.carousel.interval;
+     * ```
+     * @memberof IgxCarouselComponent
+     */
     @Input()
     get interval(): number {
         return this._interval;
     }
-
+    /**
+     *Sets the time interval in milliseconds before the slide changes.
+     *If not set, the carousel will not change slides automatically.
+     * ```html
+     * <igx-carousel interval = 1000></carousel>
+     * ```
+     * @memberof IgxCarouselComponent
+     */
     set interval(value: number) {
         this._interval = +value;
         this._restartInterval();
     }
-
+    /**
+     * Returns the value which indicates that the current slide is active.
```

Comment:
tabIndex does not indicate if the slide is active. Returns the tabIndex of the carousel element is enough as description.